test/client/fixtures/sandbox/shadow-ui-test.js

```diff
@@ -809,3 +809,16 @@ test('querySelector call should return a first non-shadowUI element (GH-1131)',
     document.head.removeChild(meta);
     document.head.removeChild(shadowMeta);
 });
+
+test('processed nodeList should have non-enumerable item and namedItem properties (GH-1141)', function () {
```

Comment:
The name of test has single quotes. Maybe do you mean wrap item and namedItem to double quote?